go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/casbin/casbin/v2
 
-require github.com/Knetic/govaluate v3.0.1-0.20171022003610-9aa49832a739+incompatible
+require (
+	github.com/Knetic/govaluate v3.0.1-0.20171022003610-9aa49832a739+incompatible
+	github.com/pkg/errors v0.9.1
```

Comment:
Why don't use the built-in error? 3rd party imports should be avoided.

Reply:
Don't worry it, `github.com/pkg/errors` can be compatible with `errors` provided by go.
 
It provides great functionality. 
such like https://godoc.org/github.com/pkg/errors#hdr-Formatted_printing_of_errors.

Reply:
But Casbin is a basic library. We don't want to import too many 3rd-party libraries, because this will bring burden to downstream users.

Currently, we only have one dependency, aka govaluate. Even model config reading is implemented without a library. I don't want to add the second dependency just for an unofficial error formatting functionality.

Reply:
Casbin users does not need to `go get github.com/pkg/errors` via bash.

Reply:
I think we should not add a 2nd dependency in go.mod file: https://github.com/casbin/casbin/blob/master/go.mod

Reply:
Resolved.